src/Workspaces/Core/Portable/Shared/TestHooks/TaskExtensions.cs

```diff
@@ -19,18 +19,26 @@ public static Task CompletesAsyncOperation(this Task task, IAsyncToken asyncToke
             return task.CompletesTrackingOperation(asyncToken);
         }
 
+        [PerformanceSensitive(
+            "https://developercommunity.visualstudio.com/content/problem/854696/changing-target-framework-takes-10-minutes-with-10.html",
+            AllowCaptures = false)]
         public static Task CompletesTrackingOperation(this Task task, IDisposable token)
         {
             if (token == null || token == EmptyAsyncToken.Instance)
             {
                 return task;
             }
 
-            return task.SafeContinueWith(
-                t => token.Dispose(),
```

Comment:
📝 This line was causing `token` to get a closure allocation even if the method returned early.